framework/organisms/nav-tabs/_nav-tabs.scss

```diff
@@ -16,12 +16,12 @@ ul.nav-tabs,
   margin-bottom: 2em !important;
   border-bottom: 2px solid $grey;
 
-  & > li {
+  > li {
     margin-bottom: -2px;
 
     &.active {
-      & > a {
-        border-bottom: 2px solid #fff;
+      > a {
+        border-bottom: 2px solid $white;
```

Comment:
We have it being applied everywhere already!
I can create a ticket on the EUTHEME board and remove them.